templates/server-ha-active-service.yaml

```
@@ -15,6 +15,11 @@ metadata:
   annotations:
 {{ template "vault.service.annotations" .}}
 spec:
+  {{- if and (.Values.server.service.externalTrafficPolicy ) (print .Values.server.service.externalTrafficPolicy | lower | title | eq "Local" ) (.Values.server.service.nodePort) (eq (.Values.server.service.type | toString) "NodePort") }}
+  externalTrafficPolicy: {{ .Values.server.service.externalTrafficPolicy| lower | title | quote }}
+  {{- else }}
+  externalTrafficPolicy: {{ default "Cluster" | quote }}
+  {{- end}}
```

I feel like it may be simpler to document the possible traffic policies in `values.yaml` and then do:

```suggestion
  {{- if .Values.server.service.externalTrafficPolicy  }}
  externalTrafficPolicy: {{ .Values.server.service.externalTrafficPolicy }}
  {{- end }}
```

---

@benashz I thought to create a "fail-proof" deployment but I'm guessing that would be easier.
I would change this to the suggested value.